src/hooks/api/useCashFlow.ts

```diff
@@ -44,12 +44,25 @@ export function useCashFlow(
               JOIN accounts AS main_account ON main_account.guid = '${account}'
               WHERE commodities.guid = main_account.commodity_guid
             ) AS mnemonic,
-            SUM(((cast(main_split.quantity_num as REAL) / main_split.quantity_denom) / (cast(main_split.value_num as REAL) / main_split.value_denom)) * (cast(splits.value_num as REAL) / splits.value_denom)) as total
+            SUM(
+              CASE
+                WHEN splits.guid = credit_split.guid THEN
+                  -1 * cast(account_split.quantity_num AS REAL) / account_split.quantity_denom
+                ELSE
+                  CASE
+                    WHEN account_split.guid = credit_split.guid THEN
+                      (cast(account_split.quantity_num AS REAL) / account_split.quantity_denom) / (cast(account_split.value_num AS REAL) / account_split.value_denom) * (cast(splits.value_num AS REAL) / splits.value_denom)
```

Comment:
When the account_split is the same as the credit_split then we pick the splits value and convert it to the account_split currency. This helps for when other splits and the account_split are in different currencies so we display the cashflow in the currency of the account we are calculating for